### PR TITLE
Support for IF

### PR DESCRIPTION
**Pull request by vladaionescu**
Re https://github.com/earthly/earthly/issues/779

### CODE REVIEW

File: earthfile2llb/converter.go [hunk 1]
```diff
@@ -352,6 +353,60 @@ func (c *Converter) RunLocal(ctx context.Context, args []string, pushFlag bool)
...
+	var opts []llb.RunOption
+	mountRunOpts, err := parseMounts(mounts, c.mts.Final.Target, c.mts.Final.TargetInput, c.cacheContext)
+	if err != nil {
+		return 0, errors.Wrap(err, "parse mounts")
+	}
+	opts = append(opts, mountRunOpts...)
+	if privileged {
+		opts = append(opts, llb.Security(llb.SecurityModeInsecure))
+	}
+	runStr := fmt.Sprintf(
```
Comment (alexcb) [inline earthfile2llb/converter.go:379]: How will this work if we're in a LOCALLY target? the RunLocal doesn't use internalRun.

File: examples/tests/if.earth [hunk 2]
```diff
@@ -0,0 +1,131 @@
+FROM alpine:3.13
```
Comment (alexcb) [inline examples/tests/if.earth:1]: I think it would be good to also try this test with LOCALLY targets.

**Review by alexcb — APPROVED** [review]
Looks good, but I think we'll have to do some additional work to support this in LOCALLY targets.

File: examples/tests/if.earth (hunk 2 above)
Comment (vladaionescu) [reply examples/tests/if.earth:1]: True... I'll need to do that in a future PR.

Comment (vladaionescu) [reply examples/tests/if.earth:1]: https://github.com/earthly/earthly/pull/794

**Pull request merged**